libs/entitlement/src/main/java/org/elasticsearch/entitlement/runtime/policy/FileAccessTree.java

```diff
@@ -138,8 +138,9 @@ private FileAccessTree(
             });
         }
 
-        // everything has access to the temp dir and the jdk
+        // everything has access to the temp dir, config dir and the jdk
         addPathAndMaybeLink.accept(pathLookup.tempDir(), Mode.READ_WRITE);
+        addPathAndMaybeLink.accept(pathLookup.configDir(), Mode.READ);
```

Comment:
Do you want this to be permanent, or a stop-gap solution until we figure out the policy files?
If it's the latter, can you add a TODO comment please?

Reply:
done